app/Http/Middleware/PasswordConfirmation.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Auth\Middleware\RequirePassword;
+use Illuminate\Validation\UnauthorizedException;
+
+class PasswordConfirmation extends RequirePassword
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @param \Closure $next
+     * @return mixed
+     * @throws \Throwable UnauthorizedException
+     */
+    public function handle($request, Closure $next, $redirectToRoute = null)
+    {
+        throw_if(! auth()->user(), new UnauthorizedException());
```

Comment:
Assuming the guard doesn't matter at this point?

Reply:
Nope, it shouldn't matter